dolphinscheduler-storage-plugin/dolphinscheduler-storage-hdfs/src/main/java/org/apache/dolphinscheduler/plugin/storage/hdfs/HdfsStorageOperator.java

```diff
@@ -77,17 +77,12 @@
 public class HdfsStorageOperator implements Closeable, StorageOperate {
 
     private static final Logger logger = LoggerFactory.getLogger(HdfsStorageOperator.class);
-    private String hdfsUser;
-    public static final String RM_HA_IDS = PropertyUtils.getString(Constants.YARN_RESOURCEMANAGER_HA_RM_IDS);
-    public static final String APP_ADDRESS = PropertyUtils.getString(Constants.YARN_APPLICATION_STATUS_ADDRESS);
-    public static final String JOB_HISTORY_ADDRESS = PropertyUtils.getString(Constants.YARN_JOB_HISTORY_STATUS_ADDRESS);
-    public static final int HADOOP_RESOURCE_MANAGER_HTTP_ADDRESS_PORT_VALUE =
-            PropertyUtils.getInt(Constants.HADOOP_RESOURCE_MANAGER_HTTPADDRESS_PORT, 8088);
+    private static HdfsStorageProperties hdfsProperties = new HdfsStorageProperties();
```

Comment:
Could we avoid instantiating `HdfsStorageProperties` here and do it in the `init()` method? `init()` method is called in both constructors. If u want to use `hdfsProperties.getKerberosExpireTime()`, what about make `getKerberosExpireTime` static?

Reply:
will take a look

Reply:
Hi @EricGao888 ,I test locally and `CacheBuilder` is before `init` call,  https://github.com/apache/dolphinscheduler/blob/b600234debd2fc01cf657b735b4864444207b502/dolphinscheduler-storage-plugin/dolphinscheduler-storage-hdfs/src/main/java/org/apache/dolphinscheduler/plugin/storage/hdfs/HdfsStorageOperator.java#L83-L92
and will throw NPE error, here